frontend/javascripts/oxalis/view/right-border-tabs/segments_tab/segments_view.tsx

```diff
@@ -681,13 +681,18 @@ class SegmentsView extends React.Component<Props, State> {
       };
     }
 
-    if (!features().jobsEnabled) {
-      title = "Computation jobs are not enabled for this WEBKNOSSOS instance.";
+    if (
+      !features().jobsEnabled ||
+      !this.props.dataset.dataStore.jobsSupportedByAvailableWorkers.includes(
```

Comment:
is it guaranteed that jobsSupportedByAvailableWorkers is empty if jobsEnabled is false? Then, we could remove the jobsEnabled check almost everywhere, right?

Reply:
(I think, I'd actually prefer this)

Reply:
I wasn't sure either. Looks like `jobsEnable == False`  leads to an empty array of supported jobs. So, I can apply your suggestion.

<img width="491" alt="image" src="https://github.com/scalableminds/webknossos/assets/1105056/7f31b0cd-38f5-4e37-bfe0-b3d7e6ad0fa1">